website/pages/docs/setup.mdx

```diff
@@ -242,183 +200,115 @@ yarn prepare
 
 Finally open `package.json` file and configure `npm run prepare` command like above.
 
-Be sure to run `npm run prepare` once you have made these changes
+Be sure to run `npm run prepare` once you have made these changes.
 
 For reference, [`ts-patch`](https://github.com/nonara/ts-patch) is an helper library of TypeScript compiler that supporting custom transformations by plugins. From now on, whenever you run `tsc` command, your `typia` function call statements would be transformed to the optimal operation codes in the compiled JavaScript files.
 
-<br/>
+{/* <br/>
 <Alert severity="warning">
   <AlertTitle>
     **`npx typia patch`** 
   </AlertTitle> 
 
-Since TypeScript v5.3 update, `tsc` no more parses `JSDocComment`s. Therefore, `typia` also cannot utilize those `JSDocComment` related features too, especially ["Comment Tags"](./validators/tags/#comment-tags) and ["JSON schema generator"](./json/schema). 
+Since TypeScript v5.3 update, `tsc` no more parses `JSDocComment`s. Therefore, `typia` also cannot utilize those `JSDocComment` related features either, especially ["Comment Tags"](./validators/tags/#comment-tags) and ["JSON schema generator"](./json/schema). 
 
 The `npx typia patch` command has been developed to revive the `JSDocComment` parsing feature of `tsc`. It is temporary solution for the TypeScript v5.3 update instead of [`ts-patch`](https://github.com/nonara/ts-patch), and will be disabled after [`ts-patch`](https://github.com/nonara/ts-patch) starts supporting such TypeScript v5.3 update.
 
 Of course, if you don't use any ["Comment Tags"](./validators/tags/#comment-tags) and ["JSON schema generator"](./json/schema), you don't need to run `npx typia patch` command. This is not mandatory command, but just optional command.
 
-</Alert>
+</Alert> */}
 
 
 
 
-## Generation
+## Bundlers
+### `unplugin-typia`
+[`unplugin-typia`](https://jsr.io/@ryoppippi/unplugin-typia) is a plugin to integrate `typia` into your bundlers seamlessly.
+
+Currently, `unplugin-typia` supports the following bundlers:
+  - [Vite](https://vitejs.dev/)
+  - [Rollup](https://rollupjs.org/)
+  - [esbuild](https://esbuild.github.io/)
+  - [webpack](https://webpack.js.org/)
+  - [Next.js](https://nextjs.org/)
+  - [Rspack](https://www.rspack.dev/)
+  - [Rolldown](https://rolldown.rs/)
+  - [farm](https://farm-fe.github.io/)
+
```

Comment:
- [bun.build](https://bun.sh/docs/bundler)

I also added a partial support for Bun.
It works for generating code, but Bun fails to solve the CJS codes including `randexp.js` (like https://github.com/oven-sh/bun/issues/7465)
So It's up to you if you include `Bun` to the official document.

I think this plugin for `Bun.build` will be the easiest way to try typia in the future, but we need to wait for Bun's stablization

Reply:
I updated a Docs.
We can use `Bun.build` fully. So please add this !